src/fusion/bgt_fuser.py

```diff
@@ -500,3 +509,151 @@ def get_label_mask(self, points, labels, mask, tilecode):
         logger.debug('Matches for [{self.bgt_type}]: ' + match_str)
 
         return label_mask
+
+
+class BGTStreetFurnitureFuser(BGTFuser):
+    """
+    Data Fuser class for automatic labelling of street furniture (point)
```

Comment:
is het "street furniture objects" of gewoon "street furniture"

Reply:
We gebruiken overal "object" dus laat maar zo staan.